jvm/sparkjl/src/main/scala/org/apache/spark/api/julia/JuliaRDD.scala

```diff
@@ -76,11 +76,11 @@ object JuliaRDD extends Logging {
 
       // Create and start the worker
       val juliaHome = sys.env.get("JULIA_HOME").getOrElse("")
-      val juliaVersion = sys.env.get("JULIA_VERSION").getOrElse("v0.5")
+      val juliaVersion = sys.env.get("JULIA_VERSION").getOrElse("v0.7")
       val juliaCommand = Paths.get(juliaHome, "julia").toString()
       val juliaPkgDir =  sys.env.get("JULIA_PKGDIR") match {
           case Some(i) => Paths.get(i, juliaVersion, "Spark").toString()
-          case None => Process(juliaCommand + " -e println(Pkg.dir(\"Spark\"))").!!.trim
+          case None => Process(juliaCommand + " -e println(abspath(joinpath(Base.find_package(\"Spark\"),\"../..\")))").!!.trim
```

Comment:
maybe something like `dirname(Base.find_package(\"Spark\"))` would be better, instead of `abspath(joinpath(Base.find_package(\"Spark\"),\"../..\")))"`

Then get rid of the `src` in the `Paths.get` call in the line below.